### DISCUSSION TITLE
Buildings disappeared

### DISCUSSION THREAD

**Discussion by Kymaro** [lloesche/valheim-server-docker#40] [General]
Just installed this docker image on my server using portainer io yesterday. Woks fine but for a reason that I cannot explain, buildings disappeared in the map. Everything is unchanged but there is no more construction on the map. 

Event if I change worlds folder by a backup one, nothing on the map.

Anyone got the same issue ?

**Comment by InB4DevOps** [comment]
Might be related to a save game bug that IronGate warned about on Twitter. Did you exit the game with Alt+F4?
Does the world backup work when you use if locally?

**Reply by Kymaro** [reply]
Yes I saw the post on Twitter. Actualy no I used the logout option on escape menu. I will try the backup localy.

**Comment by nminchow** [comment]
Did you shut down the server shortly after you built these buildings? I don't think the container is [properly handling shutdowns](https://www.reddit.com/r/valheim/comments/lbkjkd/dedicated_server_restart_gracefully/), currently.

**Reply by Kymaro** [reply]
No, I checked the container when I saw the bug but was still running since first launch yesterday.
